### DISCUSSION TITLE
AudioGridder cpu spikes

### DISCUSSION THREAD

**Discussion by sdd1061** [apohl79/audiogridder#595] [Help]
I am getting high level cpu spikes that make it impossible to mix when I insert audiogridder on a track even without a plugin loaded!
Macpro 6core 3.5 128ram 2013 trashcan
server is a macbookpro 2020 16" 64ram 
cubase 11 latest build

any suggestions are helpful

**Comment by CaBuffalo** [comment]
Hi, there,

Can you describe more in detail about the problem you have? Which plugin
causes the spike? Does it happen to all of the plugins? What's your network
settings? ... etc.

On Tue, Jun 1, 2021 at 6:30 PM sdd1061 ***@***.***> wrote:

> I am getting high level cpu spikes that make it impossible to mix when I
> insert audiogridder on a track even without a plugin loaded!
> Macpro 6core 3.5 128ram 2013 trashcan
> server is a macbookpro 2020 16" 64ram
> cubase 11 latest build
>
> any suggestions are helpful
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/595>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6BMH3LBMFYJK6LRSF3TQWCR7ANCNFSM455YHSZQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by lesbrunn** [comment]
I have tried using 6 AG instances (separate server computer) in a serious session in Pro Tools 2020.12 (Windows 10, i510600k @ 49Khz, 64GB RAM). There were too many 9173 spikes, and I had to give up. I love the idea behind AG that it's supposed to reduce CPU usage in the DAW, but Instead it causes continuous spikes in Pro Tools. The spikes stop as soon as I remove the AG instances.

I really want to use AG, but it seems a lot more work needs to be done to make it do its work smoothly without Pro Tools objecting to it. Unless of course I'm doing something very wrong...

**Comment by nolanneal** [comment]
Same here.

**Reply by lesbrunn** [reply]
> Same here.

OK. It's reassuring to know I'm not the only one.

**Reply by lesbrunn** [reply]
Do you use VE Pro?

**Comment by Henri75** [comment]
> I am getting high level cpu spikes that make it impossible to mix when I insert audiogridder on a track even without a plugin loaded!
> Macpro 6core 3.5 128ram 2013 trashcan
> server is a macbookpro 2020 16" 64ram
> cubase 11 latest build
> 
> any suggestions are helpful

you are not inserting any plugin but the problems happens without connecting to a server ? or Are you able to connect to a server ?
In the monitor are the led blinking between red/green ?
I noticed that with catalina/mac server there are some connections issues (but not on mojave with local audiogridder, or with windows servers)
